schema/dex/trades.sql

```diff
@@ -157,6 +158,51 @@ WITH rows AS (
 
         UNION
 
+        --- Kyber_V2
+        SELECT
+            evt_block_time AS block_time,
+            'Kyber' AS project,
+            NULL AS VERSION,
+            src AS token_a_address,
```

Comment:
dest should be token A so need to swap a and b for addresses and amounts

Reply:
My natural thoughts A should be source and B should be dest.  But nvm, I swapped them.